usaspending_api/etl/submission_loader_helpers/file_c.py

```diff
@@ -198,10 +181,15 @@ def _save_file_c_rows(certified_award_financial, total_rows, start_time, skipped
             "object_class": row.get("object_class"),
             "program_activity": row.get("program_activity"),
             "disaster_emergency_fund": get_disaster_emergency_fund(row),
+            "distinct_award_key": create_distinct_award_key(row),
         }
 
         save_manager.append(
             load_data_into_model(award_financial_data, row, value_map=value_map_faba, save=False, reverse=reverse)
         )
 
     save_manager.save_stragglers()
+
+
+def create_distinct_award_key(row):
+    return f"{row.get('piid') or ''}|{row.get('parent_award_id') or ''}|{row.get('fain') or ''}|{row.get('uri') or ''}"
```

Comment:
Should standardize on case here too, e.g. uppercase everything.

Reply:
fair point

Reply:
Done